sources/packages/api/src/services/student-appeal/student-appeal.service.ts

```diff
@@ -147,7 +149,13 @@ export class StudentAppealService extends RecordDataModelService<StudentAppeal>
               .getSql()})`,
           );
         }),
-      )
+      );
+    if (studentId) {
+      query.andWhere("application.student.id = :studentId", {
+        studentId,
+      });
+    }
+    const queryResult = await query
```

Comment:
I think you can use the same `query` object. no need of assigning to a new variable

Reply:
I need the query result to assign to mapFromRawAndEntities.

![image](https://user-images.githubusercontent.com/54600590/193110380-a29ecc8d-da3d-4d70-8be9-2190ed73e260.png)